FirebaseAdmin/FirebaseAdmin.Tests/Auth/Users/FirebaseUserManagerTest.cs

```diff
@@ -2228,13 +2228,12 @@ public IList<string> ListUsersResponse()
     public class EmulatorFirebaseUserManagerTest : FirebaseUserManagerTest, IDisposable
     {
         public EmulatorFirebaseUserManagerTest()
-        {
-            EnvironmentVariable.FirebaseAuthEmulatorHost = "localhost:9099";
-        }
+            => this.SetFirebaseHostEnvironmentVariable("localhost:9099");
 
         public void Dispose()
-        {
-            EnvironmentVariable.FirebaseAuthEmulatorHost = string.Empty;
-        }
+            => this.SetFirebaseHostEnvironmentVariable(string.Empty);
+
+        private void SetFirebaseHostEnvironmentVariable(string value)
+            => Environment.SetEnvironmentVariable(EnvironmentVariable.FirebaseAuthEmulatorHostName, value);
```

Comment:
Might not be a bad idea to hardcode the env variable name here as a string literal. That way if the constant value changes in the implementation, we will see a test failure here which is a good thing.

Reply:
Done